File: insight/src/main/java/info/nightscout/androidaps/plugins/pump/insight/app_layer/remote_control/CancelTBRMessage.java
package info.nightscout.androidaps.plugins.pump.insight.app_layer.remote_control;

import info.nightscout.androidaps.plugins.pump.insight.app_layer.AppLayerMessage;
import info.nightscout.androidaps.plugins.pump.insight.app_layer.Service;
import info.nightscout.androidaps.plugins.pump.insight.descriptors.MessagePriority;

public class CancelTBRMessage extends AppLayerMessage {
    public CancelTBRMessage() {
        super(MessagePriority.HIGHER, false, false, Service.REMOTE_CONTROL);
    }
}

File: insight/src/main/java/info/nightscout/androidaps/plugins/pump/insight/app_layer/remote_control/CancelTBRMessage.kt
package info.nightscout.androidaps.plugins.pump.insight.app_layer.remote_control

import info.nightscout.androidaps.plugins.pump.insight.app_layer.AppLayerMessage
import info.nightscout.androidaps.plugins.pump.insight.app_layer.Service
import info.nightscout.androidaps.plugins.pump.insight.descriptors.MessagePriority

class CancelTBRMessage : AppLayerMessage(MessagePriority.HIGHER, false, false, Service.REMOTE_CONTROL)